docs/hello-neighbour.yaml

```diff
@@ -540,153 +560,199 @@ paths:
+      - in: "body"
+        name: "body"
+        description: "Question for the church"
         required: true
+        schema:
+          $ref: "#/definitions/QuestionIn"
       responses:
         200:
-          description: "Group of matched church members"
+          description: "Question updated"
           schema:
-            $ref: "#/definitions/Group"
+            $ref: "#/definitions/Question"
+        400:
+          $ref: "#/responses/ClientError"
+        401:
+          $ref: "#/responses/Unauthenticated"
+        403:
+          $ref: "#/responses/Unauthorized"
         404:
-          description: "Group not found"
+          description: "Question not found"
           schema:
-            $ref:  "#/definitions/ErrorResponse"
-            
-  /user/bulletin: # refers to requesting user's current match group's bulletin
-    post:
+            $ref: "#/definitions/Error"
+    delete:
       tags:
-      - "member"
-      summary: "Send message"
-      description: "Post a message to the group's bulletin."
-      operationId: "sendMessage"
-      consumes:
-      - "application/json"
+      - "leader"
+      summary: "Delete question"
+      description: "Delete a question from the system."
+      operationId: "deleteQuestion"
       produces:
       - "application/json"
       parameters:
       - in: "path"
-        name: "userUuid"
-        type: "string"
-        format: "uuid"
-        required: true
-      - in: "path"
-        name: "churchUuid"
-        type: "string"
-        format: "uuid"
-        required: true
-      - in: "path"
-        name: "matchGroupUuid"
+        name: "questionUuid"
         type: "string"
         format: "uuid"
         required: true
+      responses:
+        204:
+          description: "Question deleted"
+        401:
+          $ref: "#/responses/Unauthenticated"
+        403:
+          $ref: "#/responses/Unauthorized"
+        404:
+          description: "Question not found"
+          schema:
+            $ref: "#/definitions/Error"
+
+  /register:
+    post:
+      security: []
+      summary: "Create new user account"
+      description: "Create a new user. Both church leaders and members need such an account."
+      operationId: "createUser"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
       - in: "body"
         name: "body"
-        description: "Message to send to group"
+        description: "User details for new account"
         required: true
         schema:
-          $ref: "#/definitions/MessageIn"
+          $ref: "#/definitions/UserIn"
       responses:
         200:
-          description: "Message sent"
+          description: "User account created"
           schema:
-            $ref: "#/definitions/MessageIn"
-        404:
-          description: "Church/group not found"
+            $ref: "#/definitions/User"
+        400:
+          $ref: "#/responses/ClientError"
+
+  /login:
```

Comment:
Should we combine `/profile` with `/login`? It seems like both are actions that need to be done at the same time, upon logging in a user.

Reply:
I'd leave them separate because you might want to query profile when already having a valid JWT. However, there is probably no reason why `/login` couldn't retur nthe same response as `/profile`.